service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -107,60 +129,58 @@ void shouldPopulateErrorsInResponseWhenDomainNameIsNotFound() throws Exception {
     }
 
     @Test
-    void grantAccessShouldBeCalledOnceForEachUser() throws Exception {
-        given(serviceAuthorisationApi.serviceToken(anyMap()))
-            .willReturn(SERVICE_AUTH_TOKEN);
+    void updateCaseRolesShouldBeCalledOnceForEachUser() throws Exception {
+        CallbackRequest request = createCallbackRequest();
```

Comment:
could you use callbackRequest() here instead of needing a private method?

Reply:
done :)